logservice.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/urfave/cli"
+)
+
+var VERSION string
+
+var open = os.Open
+
+func scanFile(output io.Writer, file, buildID string) error {
+	body, err := open(file)
+	if err != nil {
+		return fmt.Errorf("Unexpected error opening the file: %v", err)
+	}
+
+	scanner := bufio.NewScanner(body)
+
+	for scanner.Scan() {
+		fmt.Fprintln(output, scanner.Text())
+	}
+	if err := scanner.Err(); err != nil {
+		return fmt.Errorf("Unexpected scan error: %v", err)
+	}
+
+	return nil
+}
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "logservice"
+	app.Usage = "For now, scans a file and prints to stdout"
+	app.UsageText = "logservice [options] build-id"
+	app.Copyright = "(c) 2016 Yahoo Inc."
+
+	if VERSION == "" {
```

Comment:
If this is the Environment Variable set by incrementVersion, don't you need to get it by `os.Getenv("VERSION")`